src/Components/GoalForm/GoalForm.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { createUserGoal } from '../../actions';
+import './GoalForm.css';
+
+class GoalForm extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  handleUserInput = (event) => {
+    this.setState({ userGoal: event.target.value })
+  }
```

Comment:
We will need a method to clear the input after a user has set a goal. I think we already have an issue for this. We may also want to do some conditional rendering to have the submit button change text from "submit" to "update" so a user can update their goal.